frontend/src/component/common/DialogFormTemplate/ConfigButtons/shared.styles.tsx

```diff
@@ -12,11 +12,27 @@ const visuallyHiddenStyles = {
     whiteSpace: 'nowrap',
 };
 
+const padding = (theme: Theme) => theme.spacing(1.5);
+
+export const StyledPopover = styled(Popover)(({ theme }) => ({
+    '& .MuiPaper-root': {
+        borderRadius: `${theme.shape.borderRadiusMedium}px`,
+        paddingInline: 0,
+        paddingTop: padding(theme),
```

Comment:
This is actually a great use case for css variables, but I don't think we use them many places, so I've added it as a js const for now. I'd be very happy to use variables instead, though.